src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__pic {
+   display: block;
+   margin: 0 auto;
+   height: 134px;
+   width: 160px;
+  }
+
+  &__name {
+    color: $text-color;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    line-height: 150%;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 140%;
+    margin-top: 4px;
+    margin-bottom: 16px;
```

Comment:
not fixed:

![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/3eefe38f-622e-4245-be24-8bf219e53a7d)

again: it's better to choose one: `margin-bottom` or `margin-top` and use across all code